NEWS.md

```diff
@@ -1,7 +1,10 @@
-# annoFuse v0.90.0 (released 2020-09-18)
+# annoFuse v0.91.0 (released 2022-09-26)
 
-## New features
 
+## New features
+* Add custom type annotation that checks for required columns and maps input columns to output columns using config file
+* Fusion standardization function has the ability to accept custom type input that checks for few required columns. Config file has been made optionally and can be use to map input columns with output columns
+* Custom type was built on top of existing starfusion and arriba fusion framework. This version will continue to support both of these caller types with same implementation
```

Comment:
Eg - this would go above the previous release, like: 

```
# annoFuse v0.91.0 (released 2022-09-26)

## New features
.
.
.
.


# annoFuse v0.90.0 (released 2020-09-18)

## New features
.
.
.
```